StatBot/Models/DataModel.cs

```diff
@@ -9,6 +9,7 @@ namespace StatBot.Models
     public class DataModel
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1118)